### PR TITLE
Add autocomplete

### PR DESCRIPTION
**Pull request by wilkermichael**

### CODE REVIEW

**Comment by wilkermichael** [comment]
@lornasong I accidentally hit ready for review, this is still a draft

File: command/helpers.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package command
```
Comment (wilkermichael) [inline command/helpers.go:1]: I copied this file name from nomad https://github.com/hashicorp/nomad/blob/main/command/helpers.go 

I wanted a place for common helpers that were used in various files

File: api/task_get_test.go [hunk 2]
```diff
@@ -131,19 +131,3 @@ func TestTaskLifeCycleHandler_GetAllTasks(t *testing.T) {
 		})
 	}
 }
-
-func generateExpectedGetTasksResponse(t *testing.T, req string) oapigen.TaskResponse {
```
Comment (wilkermichael) [inline api/task_get_test.go:135]: unused anywhere

**Comment by wilkermichael** [comment]
@mkam ready for review, no rush :)

**Review by mkam — COMMENTED** [review]
Changes look good to me, but could you fix the formatting on `consul-terraform-sync -help`? The word wrapping is off and there aren't new lines between the options.

Also, I noticed a few errors on those existing help messages, if you could fix them up while you're modifying the format that would be great too! There are multiple references to`Sync` instead of `Consul-Terraform-Sync`, a typo with `extension` on `-config-dir`, and `then exits` should be `then exit` on `-inspect` and `-inspect-task`.

**Review by mkam — APPROVED** [review]
Meant to approve the PR, sorry!

**Pull request merged**